test/lsp/BUILD

```diff
@@ -30,6 +30,30 @@ cc_test(
     ],
 )
 
+cc_test(
+    name = "multithreaded_protocol_test_corpus",
```

Comment:
I wonder if there's a cleaner way to define these. I'm basically using the same build logic for all of these .cc files.